spec/cypress/integration/admin/ParticipantManagment.feature

```diff
@@ -13,7 +13,7 @@ Feature: Admin user managing participants
 
     When I type "example" into "search box"
     And I press enter in "search box"
-    Then the table should have 5 rows
+    Then the table should have 0 rows
```

Comment:
This concerns me a bit, do we know why it changed?

Reply:
@peteryates because there are 0 results for the "example" keyword. There is a similar test right beneath it with a keyword that actually returns a results.

Reply:
Ah, well deduced! Thank you @leoapost